### PR TITLE
fix: move the page view v1 enrichment to sdk and other

### PR DESCRIPTION
**Pull request by yuhao900914**

### CODE REVIEW

**Review by Mercy811 — APPROVED** [review]

File: src/amplitude-wrapper.js [hunk 1]
```diff
@@ -143,43 +143,20 @@ import { version } from '../package.json';
...
       const pageViewLegacy = configuration['pageViewLegacy'];
 
-      if (userAgentEnrichment) {
+      if (userAgentEnrichmentOptions) {
         client.add(userAgentEnrichmentPlugin(userAgentEnrichmentOptions));
       }
 
       if (pageViewLegacy) {
-        client.add(pageViewPlugin())
+        client.add(pageViewV1EnrichmentPlugin())
```
Comment (justin-fiedler) [inline src/amplitude-wrapper.js:157]: How does `pageViewV1EnrichmentPlugin()` get imported here? Is it included automatically in the GTM snippet?

Comment (yuhao900914) [reply src/amplitude-wrapper.js]: Update this to use the script to install the packages.

File: src/amplitude-wrapper.js [hunk 2]
```diff
@@ -4,6 +4,14 @@ import { version } from '../package.json';
...
...n{name:"@amplitude/plugin-page-view-v1-enrichment-browser",type:"enrichment",setup:function(e){return n(this,void 0,void 0,(function(){return r(this,(function(t){return e.loggerProvider.log("Installing @amplitude/plugin-page-view-v1-enrichment-browser."),[2]}))}))},execute:function(i){return n(e,void 0,void 0,(function(){var e,n,o,u,a,l;return r(this,(function(r){return"[Amplitude] Page Viewed"==i.event_type&&(e=i.event_properties||{},n=e["[Amplitude] Page Domain"],o=e["[Amplitude] Page Location"],u=e["[Amplitude] Page Path"],a=e["[Amplitude] Page Title"],l=e["[Amplitude] Page URL"],(i=t(t({},i),{event_type:"Page View",event_properties:t(t({},i.event_properties),{page_domain:null!=n?n:"",page_location:null!=o?o:"",page_path:null!=u?u:"",page_title:null!=a?a:"",page_url:null!=l?l:""})})).event_properties&&(delete i.event_properties["[Amplitude] Page Domain"],delete i.event_properties["[Amplitude] Page Location"],delete i.event_properties["[Amplitude] Page Path"],delete i.event_properties["[Amplitude] Page Title"],delete i.event_properties["[Amplitude] Page URL"])),[2,i]}))}))}}};return e.pageViewV1EnrichmentPlugin=i,e.plugin=i,Object.defineProperty(e,"__esModule",{value:!0}),e}({});
```
Comment (justin-fiedler) [inline src/amplitude-wrapper.js:8]: 🙌

File: src/amplitude-wrapper.js [hunk 3]
```diff
@@ -143,43 +151,20 @@ import { version } from '../package.json';
...
 
-      if (userAgentEnrichment) {
-        client.add(userAgentEnrichmentPlugin(userAgentEnrichmentOptions));
+      if (userAgentEnrichmentOptions) {
+        client.add(amplitudeUserAgentEnrichmentPlugin.userAgentEnrichmentPlugin(userAgentEnrichmentOptions));
       }
 
       if (pageViewLegacy) {
-        client.add(pageViewPlugin())
+        client.add(amplitudePageViewV1EnrichmentPlugin.pageViewV1EnrichmentPlugin())
```
Comment (justin-fiedler) [inline src/amplitude-wrapper.js:165]: nit: can we also use `amplitudePageViewV1EnrichmentPlugin.plugin` and `amplitudeUserAgentEnrichmentPlugin.plugin`? could be a little cleaner

**Review by justin-fiedler — APPROVED** [review]

File: src/amplitude-wrapper.js (hunk 3 above)
Comment (yuhao900914) [reply src/amplitude-wrapper.js:165]: @justin-fiedler 
`pageViewV1EnrichmentPlugin` is the package name for the plugin.  -> so this cannot be the same.
amplitudePageViewV1EnrichmentPlugin This is what we assign to load the plugin functions. 
How about change amplitudePageViewV1EnrichmentPlugin -> pageViewPlugin and amplitudeUserAgentEnrichmentPlugin -> userAgentPlugin?

**Pull request merged**